resources/codegen_inputs/test.rs

```diff
@@ -9,6 +9,11 @@ table KindsOfOffsets {
     /// An offset that is nullable, but always present
     #[nullable]
     nullable_offset: BigEndian<Offset16<Dummy>>,
+    /// count of the array at array_offset
```

Comment:
What's the deal with test.rs in codegen inputs? Is it a workaround for lack of a good way to test or ...?

Reply:
Maybe worth putting that explanation into the comments of test.rs or a README.md for codegen_inputs?

Reply:
It is mentioned in the codegen readme, but could definitely also be mentioned in the input file itself.